Danki2/ProjectSettings/TagManager.asset

```diff
@@ -3,7 +3,9 @@
 --- !u!78 &1
 TagManager:
   serializedVersion: 2
-  tags: []
+  tags:
+  - Enemy
+  - Friendly
```

Comment:
We're not setting this tag in the Start method of Friendly. I think we should actually just delete friendly for now. We don't really know if we'll ever need it so best to just get rid I think.